commands/daemon/daemon.go

```diff
@@ -357,10 +354,7 @@ func (s *ArduinoCoreServerImpl) LibraryInstall(req *rpc.LibraryInstallRequest, s
 		func(p *rpc.DownloadProgress) { stream.Send(&rpc.LibraryInstallResponse{Progress: p}) },
 		func(p *rpc.TaskProgress) { stream.Send(&rpc.LibraryInstallResponse{TaskProgress: p}) },
 	)
-	if err != nil {
-		return convertErrorToRPCStatus(err)
-	}
-	return stream.Send(&rpc.LibraryInstallResponse{})
+	return convertErrorToRPCStatus(err)
```

Comment:
Can a similar approach be applied to other `stream.Send()`? Like the return at the end of the `PlatformInstall` command above.
Also I think we should also add some test to avoid regression